perses/tests/utils.py

```diff
@@ -522,7 +522,7 @@ def  generate_solvated_hybrid_test_topology(current_mol_name="naphthalene", prop
         hs = [atom for atom in modeller.topology.atoms() if atom.element.symbol in ['H'] and atom.residue.name not in ['MOL','OLD','NEW']]
         modeller.delete(hs)
         modeller.addHydrogens(forcefield=system_generator.forcefield)
-        modeller.addSolvent(system_generator.forcefield, model='tip3p', padding=9.0*unit.angstroms)
+        modeller.addSolvent(system_generator.forcefield, model='tip3p', padding=20.0*unit.angstroms)
```

Comment:
Why is it 20 A here instead of 11 A as it was with the other tests?

Reply:
Good catch! I will double check that one!